src/Katas/codeImprovements/execise1.js

```diff
@@ -0,0 +1,6 @@
+function calculateAverage(numbers) {
+  const sum = numbers.reduce((accumulator, number) => accumulator + number, 0);
+  const average = sum / numbers.length;
```

Comment:
- [ ] kindly consider avoiding having unneeded variables as this variable is not changing anything on the code  😀